.github/workflows/valgrind.yml

```diff
@@ -0,0 +1,152 @@
+name: Valgrind Tests
```

Comment:
Optimize this .yml in a similar way than PR https://github.com/telefonicaid/fiware-orion/pull/3932

Reply:
Fixed in ae779cc5387faeb36c7e6a718def3b6edab24277